README.md

```diff
@@ -26,8 +26,12 @@ $ docker run -t --rm -v "$PWD":/usr/src/app -p "4000:4000" starefossen/github-pa
 
 Your Jekyll page will be available on `http://localhost:4000`.
 
-Remember to add all the gems to your `_config.yml` file in order to get all the
-different things to work correctly:
+Remember to add all the plugin gems to both your Gemfile and the
+`_config.yml` file in order to install the gems during `docker run` and to
+get all the different things to work correctly in Jekyll.
+
+*** Are there some gems brought in that users do not need to explicitly declare?
```

Comment:
@Starefossen this would be a useful addition

Reply:
Up until the last 2-3 months, we managed without a Gemfile. There were a number of authors on a mixture of Mac and PC, and some people ran Jekyll directly while others used Docker. Then a number of problems started happening, and the best we could grasp was dependency hell. So I added a Gemfile which simply had these three lines:

```
source 'https://rubygems.org'
gem 'github-pages', '160',  :group => :jekyll_plugins
gem 'tzinfo-data', :platforms => [:mingw, :mswin, :x64_mingw]
```

Interestingly, the jekyll-seo-tag gem is one of the many gems that gets installed after doing a `bundle install`. Have a look at our [Gemfile.lock](https://gist.github.com/siliconmeadow/7cd0a9e90bf7d64f772e8597cabbb5fc) which shows all the ruby gems installed.

I hope this comment might help provide some insight?